src/ThemeProvider.js

```diff
@@ -0,0 +1,89 @@
+import React, {PropTypes, Component} from 'react'
```

Comment:
I prefer to keep my files lower case and snake-cased (so this would be `theme-provider.js`. I should probably have a lint rule for that. But the reason is git has a hard time renaming files when it's purly case-changes and some filesystems are not case sensitive so it can lead to confusion.